### ISSUE TITLE
SQL Server v16 is not supported

### ISSUE THREAD

**Issue by renan-alonkin** [microsoft/mssql-django#174]
Hello there; 

I have a simple question/request. 

I want to connect to a SQL Server DB (SQL Server 2022), but after a little search, I noticed that the library doesn't support V16 yet. 

Any ideas of when it will be solved? 
Is there any workaround possible in the current state (the current version is 1.1.3)?

I checked the other issues, but they all were closed by the creators. 

Unfortunately, I can't downgrade the Database because it is already in use. 

Thanks!

**Comment by mShan0** [comment]
There is a workaround to enable v16, however this is untested and we don't know what is working and what isn't.
In the source code, go into mssql/base.py. From there, add `16: 2022` under _sql_server_versions.

```
    _sql_server_versions = {
        ....,
        15: 2019,
        16: 2022, // Add this line
    }
```

**Comment by vamshikrishnaakula** [comment]
hello renan-alonkin  am also searching quite related like this issue, may i expect how did you do dockerize the sql server 

and does this possible to decorize sql server 2012 Thankyou

**Comment by renan-alonkin** [comment]
> hello renan-alonkin am also searching quite related like this issue, may i expect how did you do dockerize the sql server
> 
> and does this possible to decorize sql server 2012 Thankyou

I don't think that there is a 2012 image for docker, unfortunately. 

My workaround was not a workaround. For now, I have a copy of our database, but on a 2017 server... The IT team wanted to kill me, but that was the only way to make it work, unfortunately. 

I hope you can find a solution

**Comment by mShan0** [comment]
Support for SQL Server 2022/v16 has been released with `mssql-django 1.2`
